psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -252,16 +252,21 @@ def scf_iterate(self, e_conv=None, d_conv=None):
     soscf_enabled = _validate_soscf()
     frac_enabled = _validate_frac()
     efp_enabled = hasattr(self.molecule(), 'EFP')
+    diis_rms = core.get_option('SCF', 'DIIS_RMS_ERROR')
 
     if self.iteration_ < 2:
         core.print_out("  ==> Iterations <==\n\n")
-        core.print_out("%s                        Total Energy        Delta E     RMS |[F,P]|\n\n" % ("   "
-                                                                                                      if is_dfjk else ""))
+        if diis_rms:
```

Comment:
Better to put this if similar to `is_dfjk` so we are not duplicating lines.

Reply:
Thanks for the hint, still learning Python...